### ISSUE TITLE
Fix VsCode Json files and pre-commit json check clash

### ISSUE THREAD

**Issue by serdardalgic** [EnterpriseDB/terraform-provider-biganimal#79]
Now we have [.vscode/launch.json](https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/main/.vscode/launch.json) and [terraform-registry-manifest.json](https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/main/terraform-registry-manifest.json) files that do not comply with our pre-commit JSON checks, namely [check-json](https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/main/.pre-commit-config.yaml#L21) and [pretty-format.json](https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/main/.pre-commit-config.yaml#L24).

Those files are exactly the same in the [terraform-provider-scaffolding repository](https://github.com/hashicorp/terraform-provider-scaffolding). And I think there is not much value in fixing their style for now. 

The launch.json file is a JSONC, an extension of JSON that allows comments. 
We can investigate if there is a way to check JSONC files with another pre-commit hook and exclude it from the JSON files. 

Long story short: 
* Our JSON pre-commit checks are failing for 2 files
* It errors in the pre-commit check only when those two files are changed, or `pre-commit run --all-files` are run. It's not a blocker for merging the PRs because of the pre-commit check. 
* We either need to exclude those two from the JSON checks or find adequate pre-commit checks that would target the Jsonc. 

In any case, this issue is trivial. Just writing it to document the current status. 

Related: #51
